Galaxy.html

```diff
@@ -0,0 +1,67 @@
+<!doctype html>
+<html lang="en">
+	<head>
+		<title>Galaxy</title>
+		<link rel="stylesheet" href="visual.css">
+		<link rel="shortcut icon" href="icon.ico" type="image/x-icon">
+	</head>
+	<body>
+	<header class="header">
+		
+		<nav class="bar">
+			<a href="#">Home</a>
+			<a href="#">About us</a>
+			<a href="#">Charity</a>
+			<a href="#">Contacts</a>
+			<p>UNKNOWN UNIVERSE</p>
+		</nav>
+		
+	</header>
+		
+		<div class="whole-universe">
+			<ul class="voating">
```

Comment:
This `ul`  is wanting to be inside semantic `aside` -> https://webref.ru/html/aside

Reply:
> This `ul` is wanting to be inside semantic `aside` -> https://webref.ru/html/aside

ok, i will change for correct element